user_docs/en/changes.md

```diff
@@ -6,7 +6,7 @@
 
 ### New Features
 * The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. (#16052, @mltony)
-* Added command to mute or unmute all other applications, not assigned to any gesture by default. (#16052, @mltony)
+* Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`. (#16052, @mltony)
```

Comment:
I think there needs to be a note associated to these that you must enable this functionality in settings first.
Also please move this to 2025.1